utils/fractribution.py

```diff
@@ -21,7 +21,7 @@
 from typing import Iterable, List, Mapping, Tuple
 
 # Default channel name when no match is found.
-UNMATCHED_CHANNEL = "Unmatched_Channel"
+# UNMATCHED_CHANNEL = "Unmatched_Channel"
```

Comment:
I've had a think about it and let's we keep Unmatched_Channel in rather than allowing the user to exclude it entirely. Having unmatched channel gives us the transparency that certain conversions couldn't be matched to a channel based on the channel classification and (if it's a larger number) encourages the user to look at and iterate their channel classification. If we allow exclusion of this it potentially 'hides' these numbers and then skews the calculation for what could be a significant channel that just hasn't been configured correctly (or at all) in the channel classification logic.

Reply:
This is a good point, but i'm not sure that keeping in unmatched_channel here achieves this. 

It gets a little awkward if the user specifies in the `channels_to_exclude` variable to remove `unmatched_channels`, as those rows are filtered out by the SQL, but then a row is added to report_table that looks like:
 | unmatched_channel | 0 | 0 | 10000 | null | null |
 
If we want to encourage the user to keep 'unmatched_channel' in, perhaps we can add a comment at the top of the channel_classification macro that encourages them to keep a catch-all such as `unmatched_channel` at the end of the case statement?

let me know if this doesn't make sense

Reply:
Yeah - I think that's the way to go. If we can add a comment along the lines of

`keep Unmatched_Channel as the default value for channels that are not matched by your case statement`

or similar.